docs/dev/new_check_howto.md

```diff
@@ -86,11 +84,11 @@ A Check is a Python class with the following requirements:
 * It must derive from `AgentCheck`
 * It must provide a method with this signature: `check(self, instance)`
 
-Checks are organized in regular Python packages under the `datadog_checks` namespace, so your code should live under `my_check/datadog_checks/my_check`. The only requirement is that the name of the package has to be the same as the check name, `my_check` in this case. The name of the Python modules within that package and the name of the class implementing the check can be whatever instead.
+Checks are organized in regular Python packages under the `datadog_checks` namespace, so your code should live under `my_check/datadog_checks/my_check`. The only requirement is that the name of the package has to be the same as the check name. There are no particular restrictions on the name of the Python modules within that package, nor the the name of the class implementing the check.
```

Comment:
typo: `, nor the the name of the class implementing the check.` / `nor on the name of the class implementing the check.`